crates/common/src/compile.rs

```diff
@@ -401,7 +401,7 @@ pub fn compile_target_with_filter(
 /// Creates and compiles a project from an Etherscan source.
 pub async fn compile_from_source(
     metadata: &Metadata,
-) -> Result<(ArtifactId, ContractBytecodeSome)> {
+) -> Result<(ArtifactId, u32, ContractBytecodeSome)> {
```

Comment:
Hmm, let's either make an alias type for `u32` or comment what the returned tuple means—I had to read the fn source to understand that it was a file_id

Reply:
Yep, good idea thanks